packages/lesswrong/lib/sql/BulkWriter.ts

```diff
@@ -0,0 +1,99 @@
+import Query from "./Query";
+import Table from "./Table";
+import InsertQuery from "./InsertQuery";
+import UpdateQuery from "./UpdateQuery";
+import DeleteQuery from "./DeleteQuery";
+
+export type BulkWriterResult = {
+  ok: number,
+}
+
+/**
+ * This class acts as a helper to facilitate the Mongo `bulkWrite` operation.
+ *
+ * It takes an arbitrary number of write operations and performs them in
+ * sequence. No guarantees are made about ordering, and performance characteristics
+ * will almost certainly be different to Mongo.
```

Comment:
I don't think there's anything that should be too problematic.

I've just pushed another change which will now send all the queries to PG in one go, so this will al least remove all the network overhead, and I trust PG to be able to do the rest fast enough once it's got the data.

Reply:
> I've just pushed another change which will now send all the queries to PG in one go

Oh excellent